### ISSUE TITLE
Add usage

### ISSUE THREAD

**Issue by peter279k** [php-api-clients/skeleton#27]
As title. I think the `README` should be enhanced so that the developers will know how to use this skeleton to build their new project :).

**Comment by WyriHaximus** [comment]
Yes it should absolutely should. Do you want to figure it out and do that, or should I put the basics up which you can enricht?

**Comment by peter279k** [comment]
Hi @WyriHaximus, thank you for your reply.

If possible, you can add some descriptions on this section firstly.

And I can also add some missed descriptions about that.

**Comment by WyriHaximus** [comment]
Sure, will do that later today :+1:

**Comment by WyriHaximus** [comment]
Done

**Comment by peter279k** [comment]
It's not worked fine.

Please look at issue #28.

Thanks.
